test/teslamate_web/controllers/car_controller_test.exs

```diff
@@ -263,6 +263,7 @@ defmodule TeslaMateWeb.CarControllerTest do
              charger_voltage: 229,
              charger_actual_current: 16,
              battery_range: 200,
+             charge_rate: 123,
```

Comment:
I wonder if 123 is a realistic charge rate

Reply:
See previous comments

Reply:
as before